### DISCUSSION TITLE
Feature Request: add pagination at the top in index

### DISCUSSION THREAD

**Discussion by majdghithan** [laravel/nova-issues#5937] [Ideas & Feature Requests]
![Screenshot 2023-10-08 at 20 47 17](https://github.com/laravel/nova-issues/assets/54972678/0fceba3e-446e-4fd2-9d8f-d1892df5710f)

**Comment by vesper8** [comment]
Yes please... been hoping for this change for years. Or at least if you could show the total number of records at the top

**Comment by andristet** [comment]
As an optional option, to have both - on the bottom and additionally to the top. Same with Create buttons which sometimes is nice to duplicate on the top as well
